pkg/controller/shared_state.go

```diff
@@ -1135,20 +1135,37 @@ func (c *SharedState) cleanUpAzVolumeAttachmentByNode(ctx context.Context, azDri
 func (c *SharedState) cleanUpAzVolumeAttachments(ctx context.Context, attachments []azdiskv1beta2.AzVolumeAttachment, cleanUp attachmentCleanUpMode, caller operationRequester) error {
 	var err error
 
+	var requestDetach = func(attachment *azdiskv1beta2.AzVolumeAttachment, patchRequired *bool) {
```

Comment:
How about name: `markDetachRequest` to align with the variable name `markCleanUp` below?